src/AspNetCore/Src/serverprocess.cxx

```diff
@@ -1189,121 +1189,84 @@ SERVER_PROCESS::SetupStdHandles(
     LPSTARTUPINFOW pStartupInfo
 )
 {
-    SECURITY_ATTRIBUTES     saAttr = {0};
     HRESULT                 hr = S_OK;
     SYSTEMTIME              systemTime;
-    STRU                    struLogFileName;
     BOOL                    fStdoutLoggingFailed = FALSE;
     STRU                    strEventMsg;
     LPCWSTR                 apsz[1];
-    STRU                    struAbsLogFilePath;
+    ASPNETCORE_CONFIG*      pConfig = NULL;
```

Comment:
I don't like that we query the config here. Most of the fields in config are passed in Initialize().

Reply:
We could pass in the configuration object but it requires change amount of code. Since we already have the request object, it's arguable that passing is better than querying.